README.md

```diff
@@ -67,6 +67,10 @@ Returns an array of observations to the given `callback`. Any errors are
 propagated as the first argument to the callback. Same options accepted as
 `mapeo.observationStream`.
 
+Valid `opts` include:
+
+- `opts.removeForks` - Whether to only return the observation with the newest timestamp, if multiple heads exist for an observation ID. Defaults to `false`.
```

Comment:
Could this default to true? I can imagine that most impl. would want this to be true, unless implementing a diff view

Reply:
On [osm-p2p-server](https://github.com/digidem/osm-p2p-server/blob/master/api/get_map.js) the option is called `forks` and it defaults to `false` (e.g. it defaults to only sending the most recent fork). I think it would be good to have consistency and use the same option here.

Reply:
I'm cool either way! :sunglasses: